contents/handbook/engineering/developing-locally.md

```diff
@@ -151,9 +135,12 @@ This is it – you should be seeing the PostHog app at <a href="http://localhost
 
 You can now change PostHog in any way you want. See [Project structure](/handbook/engineering/project-structure) for an intro to the repository's contents. To commit changes, create a new branch based on `master` for your intended change, and develop away.
 
-### Manual setup
+### Manual setup (advanced)
```

Comment:
I'm in favor of not having this here at all :). If you wanna do it manually, then you should own it rather than be able to point to the handbook and say "but the handbook told me I could use it!"

Reply:
I'm actually also in favor of that... one less path to worry about. I was hesitant though because of these tips and tricks for any troubles.

Reply:
@pauldambra FYI because you approved it and now I've changed more 🙈